core/src/main/java/hudson/model/Slave.java

```diff
@@ -306,6 +306,8 @@ public void setLabelString(String labelString) throws IOException {
         return new GetClockDifference1();
     }
 
+    @Override
+    @Nonnull
```

Comment:
This is not true in the case of [`GoogleSlave`](https://github.com/jenkinsci/gcloud-plugin/blob/4b84dbdd2532dc65feb5bef11f82f713360622b2/src/main/java/com/cloudbees/jenkins/plugins/gcloud/GoogleSlave.java#L19-L21), from a quick search.